### PR TITLE
#597 - Test RepositoriesFromStorage

### PR DESCRIPTION
**Pull request by genryxy**
Part of #597 
Added tests for `RepositoriesFromStorage`

### CODE REVIEW

File: src/test/java/com/artipie/RepoConfigYaml.java [hunk 1]
```diff
@@ -68,6 +68,16 @@ public RepoConfigYaml withFileStorage(final Path path) {
...
+    /**
+     * Adds aliase storage to config.
+     * @param aliase Storage aliase
+     * @return Itself
+     */
+    public RepoConfigYaml withStorageAliase(final String aliase) {
+        this.builder = this.builder.add("storage", "default");
+        return this;
+    }
+
```
Comment (olegmoz) [inline src/test/java/com/artipie/RepoConfigYaml.java:80]: @genryxy there is typo, the word is spelled "alias"

File: src/test/java/com/artipie/RepositoriesFromStorageTest.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+     */
+    private static final String REPO = "my-repo";
+
+    /**
+     * Type repository.
+     */
+    private static final String TYPE = "maven";
+
+    @Test
+    void findRepoSettingAndCreateRepoConfigWithStorageAliase() {
```
Comment (olegmoz) [inline src/test/java/com/artipie/RepositoriesFromStorageTest.java:54]: @genryxy typo in word "alias" occurs here as well. Please fix it across the PR

File: src/test/java/com/artipie/RepositoriesFromStorageTest.java [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for {@link RepositoriesFromStorage}.
+ *
+ * @since 0.14
+ */
+final class RepositoriesFromStorageTest {
```
Comment (olegmoz) [inline src/test/java/com/artipie/RepositoriesFromStorageTest.java:41]: @genryxy I think some negative cases are missing: such as missing config YAML, malformed config YAML, missing aliases config, unknown alias. These may be added in another PR, but then this PR should not close the issue.

**Review by olegmoz — CHANGES_REQUESTED** [review]
@genryxy thanks, please see some comments

**Comment by genryxy** [comment]
@olegmoz thanks, fixed typo. I'll add negative test cases in another PR so I returned todo

**Review by olegmoz — APPROVED** [review]
@genryxy thanks, merging
